### PR TITLE
cubiomes-viewer: Add version 2.6.1

### LINKED ISSUE #10146: [Request]: cubiomes-viewer
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 100 stars and/or 50 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

cubiomes-viewer

### Description

An efficient graphical Minecraft seed finder and map viewer.

### Homepage

https://github.com/Cubitect/cubiomes-viewer

### Download Link(s)

https://github.com/Cubitect/cubiomes-viewer/releases/

### Some Indication of Popularity/Repute

GitHub stars: ~400

### PR DESCRIPTION
**Pull request by THEshezzee**
Closes #10146

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by HUMORCE** [comment]
> If the application provides only a 32bit download, the architecture field is not required. In all other cases, architecture field is mandatory.
>
> [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md)

**Review by HUMORCE — CHANGES_REQUESTED** [review]
x64

**Comment by THEshezzee** [comment]
/verify
> x64

Sorry, resolved https://github.com/ScoopInstaller/Extras/pull/10147/commits/fdc573bbd84a69dea51c3a8fac89e6e9249c1ed0

File: bucket/cubiomes-viewer.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+    "version": "3.0.0",
+    "description": "An efficient graphical Minecraft seed finder and map viewer.",
+    "homepage": "https://github.com/Cubitect/cubiomes-viewer",
+    "license": "GPL-3.0",
```
Comment (HUMORCE) [inline bucket/cubiomes-viewer.json:5]:
```suggestion
    "license": "GPL-3.0-or-later",
```

**Review by HUMORCE — APPROVED** [review]

File: bucket/cubiomes-viewer.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            "hash": "8b715b8f153ded3d08d36af3bf04168041eeff79e5b03322e072262236596237",
+            "shortcuts": [
+                [
+                    "cubiomes-viewer-3.0.0-w64.exe",
+                    "Cubiomes Viewer"
+                ]
+            ]
+        }
+    },
+    "checkver": "github",
```
Comment (HUMORCE) [inline bucket/cubiomes-viewer.json:18]:
```suggestion
    "architecture": {
        "64bit": {
            "url": "https://github.com/Cubitect/cubiomes-viewer/releases/download/3.0.0/cubiomes-viewer-3.0.0-w64.exe#/cubiomes-viewer.exe",
            "hash": "8b715b8f153ded3d08d36af3bf04168041eeff79e5b03322e072262236596237"
        }
    },
    "shortcuts": [
        [
            "cubiomes-viewer.exe",
            "Cubiomes Viewer"
        ]
    ],
    "checkver": "github",
```

File: bucket/cubiomes-viewer.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+                "url": "https://github.com/Cubitect/cubiomes-viewer/releases/download/$version/cubiomes-viewer-$version-w64.exe",
+                "shortcuts": [
+                    [
+                        "cubiomes-viewer-$version-w64.exe",
+                        "Cubiomes Viewer"
+                    ]
+                ]
+            }
+        }
+    }
```
Comment (HUMORCE) [inline bucket/cubiomes-viewer.json:31]:
```suggestion
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/Cubitect/cubiomes-viewer/releases/download/$version/cubiomes-viewer-$version-w64.exe#/cubiomes-viewer.exe"
            }
        }
    }
```

**Review by HUMORCE — CHANGES_REQUESTED** [review]
fix

**Review by HUMORCE — APPROVED** [review]

**Pull request merged**